test/game_rules_test.exs

```diff
@@ -0,0 +1,29 @@
+defmodule GameRulesTest do
+  use ExUnit.Case
+
+  import Mox
+
+  setup :verify_on_exit!
+
+  test "a game is over if a bomb has been selected" do
+    randomizer =
+      MockRandomizer
+      |> expect(:random_coordinate_pair, fn _, _ -> {0, 0} end)
+      |> expect(:random_coordinate_pair, fn _, _ -> {1, 0} end)
+      |> expect(:random_coordinate_pair, fn _, _ -> {2, 0} end)
+      |> expect(:random_coordinate_pair, fn _, _ -> {3, 0} end)
+      |> expect(:random_coordinate_pair, fn _, _ -> {4, 0} end)
+      |> expect(:random_coordinate_pair, fn _, _ -> {5, 0} end)
+      |> expect(:random_coordinate_pair, fn _, _ -> {6, 0} end)
+      |> expect(:random_coordinate_pair, fn _, _ -> {7, 0} end)
+      |> expect(:random_coordinate_pair, fn _, _ -> {8, 0} end)
+      |> expect(:random_coordinate_pair, fn _, _ -> {9, 0} end)
+
```

Comment:
I've seen the above `MockRandomizer` setup more than once in this PR (i.e. `BombPlacerTest` & `GameTest`), is there a way to abstract this out into a helper function and pass in the values instead of hardcoding them? If something changes in the game and one day you want to place more bombs for example, you'll have to touch this code in a few places.

Reply:
Yes, good idea! I'll do that :)